src/TesApi.Web/appsettings.json

```diff
@@ -5,26 +5,33 @@
     }
   },
   "AllowedHosts": "*",
-
+  "BatchImagePublisher": "microsoft-dsvm",
+  "BatchImageOffer": "ubuntu-hpc",
+  "BatchImageSku": "2004",
+  "BatchImageVersion": "latest",
+  "BatchNodeAgentSkuId": "batch.node.ubuntu 20.04",
+  "DisableBatchNodesPublicIpAddress": "True",
+  "DisableBatchScheduling": "False",
+  "UsePreemptibleVmsOnly": "false",
+  "UseLegacyBatchImplementationWithAutopools": "false",
+  "Gen2BatchNodeAgentSkuId": "batch.node.ubuntu 20.04",
+  "Gen2BatchImageOffer": "ubuntu-hpc",
+  "Gen2BatchImagePublisher": "microsoft-dsvm",
+  "Gen2BatchImageSku": "2004",
+  "Gen2BatchImageVersion": "latest",
+  "Gen1BatchImageOffer": "ubuntu-server-container",
+  "Gen1BatchImagePublisher": "microsoft-azure-batch",
+  "Gen1BatchImageSku": "microsoft-azure-batch",
+  "Gen1BatchImageVersion": "latest",
+  "BatchPoolRotationForcedDays": "7",
   "BatchAccount": {
-    "AccountName": "",
-    "BaseUrl": "",
-    "AppKey": "",
-    "Region": "",
-    "SubscriptionId": "",
-    "ResourceGroup": ""
+    "AccountName": ""
   },
   "CosmosDb": {
-    "CosmosDbKey": "",
-    "CosmosDbEndpoint": "",
     "AccountName": ""
   },
   "RetryPolicyOptions": {
     "MaxRetryCount": 5,
     "ExponentialBackOffExponent": 2
-  },
-  "ContainerRegistry": {
```

Comment:
Just checking you really meant to remove this?  if so, you can ignore this comment